### PR TITLE
A few Maid related changes

### PR DESCRIPTION
**Pull request by kannthus**
## About The Pull Request

We have a few people in the community who would appreciate this availability of the maid outfit. Added to both contraband for the janitors wardrobe and the petey pocket store

## Why It's Good For The Game

Might increase the chances of certain people playing Janitor, but also the Janidrobe needs more contraband anyway, as it currently has zilch. (also more purchasable options in the petey store is always a good bennefit)

## Changelog
:cl:
add: Maid Uniform, White Gloves and Laceup Shoes to the JaniDrobe as contraband.
add: Maid Uniform to the Loadout for 5000 points.
/:cl:

### CODE REVIEW

File: code/modules/client/loadout/loadout_uniform.dm [hunk 1]
```diff
@@ -256,6 +256,10 @@
 	path = /obj/item/clothing/under/costume/swagoutfit
 	cost = 69000
 
+/datum/gear/uniform/misc/janimaid
```
Comment (Zonespace27) [inline code/modules/client/loadout/loadout_uniform.dm:259]: add a donkstation comment or move to the /donkstation/ folder as per #72

File: code/modules/vending/wardrobes.dm [hunk 2]
```diff
@@ -342,6 +342,10 @@
 					/obj/item/clothing/shoes/galoshes = 2,
 					/obj/item/watertank/janitor = 1,
 					/obj/item/storage/belt/janitor = 2)
+	contraband = list(/obj/item/clothing/under/rank/civilian/janitor/maid = 1,
```
Comment (Zonespace27) [inline code/modules/vending/wardrobes.dm:345]: Same here

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/client/loadout/loadout_uniform.dm (hunk 1 above)
Comment (kannthus) [reply code/modules/client/loadout/loadout_uniform.dm:260]: Is that a good enough distrinction?

**Review by Zonespace27 — APPROVED** [review]
LGTM

**Pull request merged**